services/src/main/java/org/keycloak/authentication/authenticators/broker/IdpReviewProfileAuthenticator.java

```diff
@@ -107,7 +107,6 @@ protected boolean requiresUpdateProfilePage(AuthenticationFlowContext context, S
     @Override
     protected void actionImpl(AuthenticationFlowContext context, SerializedBrokeredIdentityContext userCtx, BrokeredIdentityContext brokerContext) {
         EventBuilder event = context.getEvent();
-        //velias: looks like UPDATE_PROFILE event is not fired. IMHO it should not be fired here as user record in keycloak is not changed, user doesn't exist yet 
```

Comment:
this comment is not revlevant anymore, because now `UPDATE_PROFILE` is triggered